bpf/proxy/syncer.go

```diff
@@ -967,6 +967,11 @@ type serviceInfo struct {
 	onlyNodeLocalEndpoints   bool
 }
 
+// TopologyKeys is part of ServicePort interface.
+func (info *serviceInfo) TopologyKeys() []string {
+	panic("NOT IMPLEMENTED")
```

Comment:
Probably need to understand/implement this one now.

Reply:
done